Add 'opentelekomcloud_ims_image_v2' resource and docs

ShiChangkuo opened the pull request:
The changes are as follows:
1. pull github.com/huaweicloud/golangsdk;
2. define the 'opentelekomcloud_ims_image_v2' resource;
3. add test case for TestACC;
4. add docs.

niuzhenguo commented on opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go:
```diff
@@ -0,0 +1,293 @@
+	config := meta.(*Config)
+	ims_Client, err := config.imageV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenTelekomCloud EVS storage client: %s", err)
```
should be IMS client here.

ShiChangkuo replied:
@niuzhenguo Thanks for your comments, please review it in the new commits.

niuzhenguo commented on opentelekomcloud/resource_opentelekomcloud_images_ims_v2.go:
```diff
@@ -0,0 +1,293 @@
+	log.Printf("[DEBUG] Retrieved Image %s: %#v", d.Id(), img)
+
+	d.Set("name", img.Name)
+	d.Set("id", img.ID)
```
id should be set by d.SetId(img.ID)

ShiChangkuo replied:
ok, it has been fixed.

niuzhenguo commented on website/docs/r/images_ims_v2.html.markdown:
```diff
@@ -0,0 +1,97 @@
+
+Manages a V2 Image resource within OpenTelekomCloud IMS.
+
+## Example Usage
```
better to also give an example with OBS here.

ShiChangkuo replied:
@niuzhenguo Thanks for your comments, please review it in the new commits.

niuzhenguo commented on opentelekomcloud/provider.go:
```diff
@@ -250,6 +250,7 @@ func Provider() terraform.ResourceProvider {
 			"opentelekomcloud_identity_group_v3":                  resourceIdentityGroupV3(),
 			"opentelekomcloud_identity_group_membership_v3":       resourceIdentityGroupMembershipV3(),
 			"opentelekomcloud_images_image_v2":                    resourceImagesImageV2(),
+			"opentelekomcloud_images_ims_v2":                      resourceImagesImsV2(),
```
how about naming it as opentelekomcloud_ims_image_v2, just like ecs_instance_v1, evs_volume_v3 , SERVICE_RESOURCE pattern.

ShiChangkuo replied:
yep, it looks better and I will rename the resource to opentelekomcloud_ims_image_v2

niuzhenguo reviewed (commented):
Thanks for working on this, please see comments inline.

ShiChangkuo committed:
Add 'opentelekomcloud_ims_image_v2' resource

1. pull github.com/huaweicloud/golangsdk;
2. define the 'opentelekomcloud_ims_image_v2' resource;
3. add test case for TestACC.

Signed-off-by: ShiChangkuo <shichangkuo90@gmail.com>

ShiChangkuo committed:
Add 'opentelekomcloud_ims_image_v2' docs

Signed-off-by: ShiChangkuo <shichangkuo90@gmail.com>

niuzhenguo committed:
Merge branch 'master' into feature_ims
